source/_integrations/energy.markdown

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Redirect does not seem to work https://deploy-preview-20232--home-assistant-docs.netlify.app/integrations/energy/

Reply:
Correct, you need to force the redirect in this case

Reply:
Hmm that seems only possible using a netlify configuration file

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Reply:
Hmm that seems only possible using a netlify configuration file

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Reply:
`forced` is only for that file, `301!`  should work <https://www.netlify.com/blog/2020/04/07/creating-better-more-predictable-redirect-rules-for-spas/>

Reply:
Working as expected now :+1:

Reply:
TIL! Nice! That should be in their docs 😬